test/metrics/metrics_test.go

```diff
@@ -187,6 +189,8 @@ func TestMetricsWhenUsersAutomaticallyApprovedAndThenDeactivated(t *testing.T) {
 	hostAwait := awaitilities.Host()
 	memberAwait := awaitilities.Member1()
 	memberAwait2 := awaitilities.Member2()
+	spaceprovisionerconfig.UpdateForCluster(t, hostAwait.Awaitility, memberAwait.ClusterName, testSpc.Enabled(true), testSpc.MaxNumberOfSpaces(1))
+	spaceprovisionerconfig.UpdateForCluster(t, hostAwait.Awaitility, memberAwait2.ClusterName, testSpc.Enabled(true), testSpc.MaxNumberOfSpaces(1))
```

Comment:
why do we need it here? We don't rely on the limits in this test, do we? :upside_down_face: